bundles/org.openhab.binding.goecharger/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -205,14 +205,12 @@
 		<label>Temperature type 2 port</label>
 		<description>Temperature of the Go-eCharger on the type 2 port</description>
 		<state pattern="%d %unit%" readOnly="true"/>
-		<category>Temperature</category>
```

Comment:
This should not be necessary to remove. XSD definition:

```
    <xs:complexType name="channelType">
        <xs:sequence>
            <xs:element name="item-type" type="xs:string" minOccurs="0"/>
            <xs:element name="kind" type="xs:string" minOccurs="0"/>
            <xs:element name="label" type="xs:string"/>
            <xs:element name="description" type="xs:string" minOccurs="0"/>
            <xs:element name="category" type="xs:string" minOccurs="0"/>
            <xs:element name="tags" type="thing-description:tags" minOccurs="0"/>
            <xs:element name="state" type="thing-description:state"/>
            <xs:element name="command" type="thing-description:command"/>
            <xs:element name="event" type="thing-description:event" minOccurs="0"/>
            <xs:element name="autoUpdatePolicy" type="thing-description:auto-update-policy" minOccurs="0"/>
            <xs:choice minOccurs="0">
                <xs:element name="config-description" type="config-description:configDescription"/>
                <xs:element name="config-description-ref" type="config-description:configDescriptionRef"/>
            </xs:choice>
        </xs:sequence>
        <xs:attribute name="id" type="config-description:idRestrictionPattern" use="required"/>
        <xs:attribute name="advanced" type="xs:boolean" default="false" use="optional"/>
        <xs:attribute name="system" type="xs:boolean" default="false" use="optional"/>
    </xs:complexType>
```

Can you please try to swap the lines with state and category, so they are in the correct order?